### PR TITLE
Added Blog (Interview)

### PR DESCRIPTION
**Pull request by saarthakchats**
Interview with Samyak and Vatsal -- Two Open Source contributors.

### CODE REVIEW

**Comment by abishekvashok** [comment]
Pls make sure the image is 240x240 px

**Comment by saarthakchats** [comment]
@abishekvashok 
Is it fine now?

**Comment by saarthakchats** [comment]
Sorry for closing it (by mistake).

File: _data/blogs.yml [hunk 1]
```diff
@@ -67,3 +67,8 @@
   title: Why am I doing it? I hope it will be worth at the end… (my struggles with Hugo framework)
   img: 9.jpg
   author_name: Kuba Michalski  
+
+- url: https://saarthakchaturvedi05.wordpress.com/2017/12/10/interview-with-two-open-source-contibutors/
+  title: Interview with 2 FOSSASIA ontibutors
+  img: suups.png
```
Comment (vaibhavsingh97) [inline _data/blogs.yml]: Can you please rename the pic to `10.png`.

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]

**Comment by saarthakchats** [comment]
@vaibhavsingh97 
Done

**Comment by abishekvashok** [comment]
Lets wait for Travis.

**Comment by vaibhavsingh97** [comment]
@saarthakchats Please share deployment link.

**Comment by saarthakchats** [comment]
https://heuristic-volhard-32af86.netlify.com/

I am still not sure how to release deployment link

**Comment by kubami9** [comment]
@saarthakchats you can do it by setting GitHub pages in your forked repository options 😃

**Comment by saarthakchats** [comment]
@kubami9 
I still don't understand how

**Comment by kubami9** [comment]
@saarthakchats you can go to your repo options and after scrolling down a little bit you have GitHub pages options, you gotta make sure it's enabled and give us the link :)

**Comment by saarthakchats** [comment]
https://saarthakchats.github.io/deployed-gungci17.fossasia.org/

**Review by anshumanv — APPROVED** [review]

**Comment by anshumanv** [comment]
Please resolve conflicts and this should be okay.

File: _data/blogs.yml [hunk 2]
```diff
@@ -72,6 +72,12 @@
   title: Why am I doing it? I hope it will be worth at the end… (my struggles with Hugo framework)
   img: 9.jpg
   author_name: Kuba Michalski  
+
```
Comment (anshumanv) [inline _data/blogs.yml:75]: Remove this extra line please.

**Comment by saarthakchats** [comment]
@anshumanv 
I think its fine now

**Comment by saarthakchats** [comment]
@anshumanv 
Removed the line

**Pull request merged**